### ISSUE TITLE
Latest master version does not work

### ISSUE THREAD

**Issue by max1c** [123andy/redcap-docker-compose#61]
I get the following error during the installation:

Unable to obtain installation SQL from http://localhost/install.php which is exposed to you as http://localhost/install.php

![image](https://user-images.githubusercontent.com/11587156/159061703-69508cb0-c634-432a-a7a1-cf14fa9eba1d.png)

I checked out the last commit from Sep 3, 2021 https://github.com/123andy/redcap-docker-compose/commit/1556295d861343d2a0fa4b2a9e1d1b569120c591 and it's working. 

Looks like this is the commit that broke https://github.com/123andy/redcap-docker-compose/commit/663082e00c47e49deaf66351be91939acbb64ee0

Also, please increase the default upload size to ~60mb. The latest redcap zips are anywhere from 50-60mb while the default upload size is <50mb.

**Comment by nsonanh** [comment]
I had this error and it's due to wrong installation file path.

In my case I can see this folder after extracting the zip file:
![image](https://user-images.githubusercontent.com/15029492/167426882-92c2b786-edc8-4aa5-b0e0-978fd14a2d1b.png)
and inside it is:
![image](https://user-images.githubusercontent.com/15029492/167426996-c0bfd830-50da-43af-b2df-575f224a7e05.png)

So after I got the docker containers running, I updated `www/REDCapInstaller.php` file:
![image](https://user-images.githubusercontent.com/15029492/167427248-275f0e1e-ddc7-4d8c-aaba-038907711b65.png)

And everything worked correctly:
![image](https://user-images.githubusercontent.com/15029492/167435301-eb11cb1e-5398-4322-adcb-92631c34acfe.png)

Hope it can help.

**Comment by 123andy** [comment]
@nsonanh - Most people place what you have as 'redcap' as the webroot for local dev.  We never envisioned having the installer be two branches down.    At least that is what I think you're saying...  

@max1c - are you still having issues?  If you want to make a pull request to address them that would be great.

**Comment by nsonanh** [comment]
@123andy Sorry if I didn't explain it clearer, I meant the path of the `redcap` folder inside the zip file.
If the zip file structure after extracted does not contain `redcap` folder as root, the web app won't be able to copy the content to `www` folder. Hence error mentioned by @max1c.

For the webroot I left it to be `/` as default without change.

**Comment by 123andy** [comment]
Did you use the zip file from the consortium?   I just tested with the latest distro and it worked as expected:

```REDCAP_WEBROOT_PATH=/```

![image](https://user-images.githubusercontent.com/7461553/167455630-3eb606aa-66fb-4f0a-9289-583a15dac21e.png)

I guess if you manually re-zip a download and it embeds it in another folder it may not work seamlessly.

This also confirms that the file upload issue reported earlier has been fixed.
